src/js/rtc_session.js

```diff
@@ -477,13 +477,82 @@ export default class RtcSession {
     get sessionReport() {
         return this._sessionReport;
     }
-
     get callId() {
         return this._callId;
     }
     get mediaStream() {
         return this._userAudioStream;
     }
+    get localVideoStream() {
```

Comment:
1. should also include the stream if it's provided by client
2. could you add document for this and mediaStream to clearly define what it gets (it's confusing when we have mediaStream and localVideoStream at the same time)